cosmos/scheduler/cmd/scheduler.go

```diff
@@ -37,6 +39,11 @@ func (s *Scheduler) Run() {
 		log.Println(err)
 		return
 	}
+	err = s.configureIngestionSecret(true)
+	if err != nil{
+		log.Fatal("error configuring secret:", err)
+	}
+	s.healthCheck()
```

Comment:
`healthCheck` is probably blocking so a goroutine would be warranted here

Reply:
Oops yeah looks like my rebase got messed up. Thank you.